lib/pages/recipes_page.dart

```diff
@@ -0,0 +1,223 @@
+          itemBuilder: (BuildContext ctxt, int index) {
+            return Dismissible(
+              confirmDismiss: (DismissDirection direction) async {
+                return await buildDeleteShowDialog(context);
+              },
+              background: Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Container(
+                  decoration: BoxDecoration(
+                    color: RED,
+                    borderRadius: BorderRadius.all(
+                      Radius.circular(5),
+                    ),
+                  ),
+                  child: Align(
+                    alignment: Alignment.centerLeft,
+                    child: Padding(
+                        padding: const EdgeInsets.only(left: 8.0),
+                        child: Icon(
+                          Icons.delete,
+                          color: WHITE,
+                        )),
+                  ),
+                ),
+              ),
+              secondaryBackground: Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Container(
+                  decoration: BoxDecoration(
+                    color: RED,
+                    borderRadius: BorderRadius.all(
+                      Radius.circular(5),
+                    ),
+                  ),
+                  child: Align(
+                    alignment: Alignment.centerRight,
+                    child: Padding(
+                        padding: const EdgeInsets.only(right: 8.0),
+                        child: Icon(
+                          Icons.delete,
+                          color: WHITE,
+                        )),
+                  ),
+                ),
+              ),
+              key: UniqueKey(),
+              child: buildRecipe(recipes, index),
+              onDismissed: (direction) {
+                _recipesProvider.deleteRecipe(recipes.keys.toList()[index]);
+              },
+            );
+          }),
+    );
+  }
+
+  Padding buildRecipe(Map<String, dynamic> recipes, int index) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: Container(
+        decoration: BoxDecoration(
+          color: _colorsApp.greyColor,
+          borderRadius: BorderRadius.all(
+            Radius.circular(5),
+          ),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: ListTile(
+            title: Text(recipes.values.toList()[index]["label"]),
+            leading: Icon(
+              Icons.receipt,
+              size: 32.0,
+            ),
+            trailing: IconButton(
+              icon: Icon(
+                Icons.edit,
+                color: BLACK,
+              ),
+              onPressed: () => openRecipePage(recipes.keys.toList()[index]),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  emptyRecipes() {
+    return SingleChildScrollView(
+      child: Center(
+        child: Column(
```

Comment:
remove Containers and MediaQuery size. Just Let Image, Icon and Text and and mainAxisAlignment.spaceEvenly to Column